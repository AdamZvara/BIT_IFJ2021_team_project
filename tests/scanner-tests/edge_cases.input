27nazdar
.else
~while

-- decimal numbers
-- no whole part
.5
-- no decimal part
1.
-- no exponent
1eabc
1.5e
1.8e-
0.0e+

-- strings
-- non existing escape sequence
-- 2 errors because error when "\p and "\n(actual new line not escaped)
"\p"     
"breakpoint"
"\1"
"\12"
-- 2 errors again, parsing "\269 and "\n separately
"\269"
"\25a"