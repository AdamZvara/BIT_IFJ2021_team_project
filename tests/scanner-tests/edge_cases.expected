TOK_INT : 27
TOK_ID : nazdar
ERROR
TOK_KEYWORD : else
ERROR
TOK_KEYWORD : while
ERROR
TOK_INT : 5
ERROR
ERROR
TOK_ID : abc
ERROR
ERROR
ERROR
ERROR
ERROR
TOK_STRING : breakpoint
ERROR
ERROR
ERROR
ERROR
ERROR
ERROR
